resources/clj-kondo.exports/nedap/speced.def/hooks/speced/def.clj

```diff
@@ -0,0 +1,17 @@
+(ns hooks.speced.def
+  (:require
+   [clj-kondo.hooks-api :as api]))
+
+(defn defprotocol-hook [{{:keys [children] :as node} :node}]
+  (let [[_macro protocol-name docstring & methods] children]
+    {:node (api/list-node ;; defprotocol with renamed fn names
```

Comment:
effectively unwraps
```clj
(speced/defprotocol Example
 ""
 (test [_])))
```

to 
```clj

(defprotocol Example
 ""
 (test [_])
 (--test [_])))
```

Which is not equal to what the lib is doing (the lib creates a defn for the `test` fn, and a `--test` protocol fn; but for the analyser that difference doesn't matter, greatly simplifying the hook :)